.config/typos.toml

```
@@ -0,0 +1,60 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+
+[files]
+extend-exclude = [
+    "deps/",
+    "src/crc16_slottable.h",
+    "src/eval.c",
```

I would like to document why these two files are excluded:
```suggestion
    # crc16_slottable includes a bunch of pre-generated random strings.
    "src/crc16_slottable.h",
    # I don't know why eval.c is excluded, so would be good to put something here.
    "src/eval.c",
```

---

It is vendored code.
https://github.com/valkey-io/valkey/blob/af1b0de92de2ace4a01e74cf9ed7d7d6f3dedf70/src/eval.c#L2

---

Well, that's more of a statement that our copyrights aren't accurate. The code was heavily modified in the last few years while we worked on it in Redis. https://github.com/valkey-io/valkey/commits/unstable/src/eval.c It was modified within the last 3 weeks.

---

I've added the regex exception needed to allow `eval.c` to be checked; and added a comment on `crc16_slottable.h`